crates/depguard-inline-suppressions/CLAUDE.md

```
@@ -0,0 +1,87 @@
+# CLAUDE.md — depguard-inline-suppressions
+
+## Purpose
+
+Inline suppression parsing for depguard. Parses `# depguard: allow(...)` comments in Cargo.toml files to suppress specific findings.
+
+## Supported Syntax
+
+### Inline comment on same line as dependency
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)